other/test_db.sql

```diff
@@ -0,0 +1,261 @@
+-- phpMyAdmin SQL Dump
+-- version 4.7.1
+-- https://www.phpmyadmin.net/
+--
+-- Host: sql3.freesqldatabase.com
+-- Generation Time: Jan 11, 2020 at 08:32 PM
+-- Server version: 5.5.54-0ubuntu0.12.04.1
+-- PHP Version: 7.0.33-0ubuntu0.16.04.3
+
+SET SQL_MODE = "NO_AUTO_VALUE_ON_ZERO";
+SET AUTOCOMMIT = 0;
+START TRANSACTION;
+SET time_zone = "+00:00";
+
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+/*!40101 SET NAMES utf8mb4 */;
+
+--
+-- Database: `sql3317654`
+--
+
+-- --------------------------------------------------------
+
+--
+-- Table structure for table `Comment`
+--
+
+CREATE TABLE `Comment` (
+  `commentId` int(11) NOT NULL,
+  `planId` int(11) NOT NULL,
+  `userId` int(11) NOT NULL,
+  `time` time NOT NULL,
+  `text` varchar(100) NOT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+
+-- --------------------------------------------------------
+
+--
+-- Table structure for table `Course`
+--
+
+CREATE TABLE `Course` (
+  `courseId` int(11) NOT NULL,
+  `credits` int(11) NOT NULL,
```

Comment:
Suggestion: `Course.credits` has at most 2 digits, so `int(2)` or `int(3)` fits this field better.